initializr-service/application.yml

```diff
@@ -31,6 +31,12 @@ initializr:
           - versionRange: 1.3.2.BUILD-SNAPSHOT
             version: Brixton.BUILD-SNAPSHOT
             repositories: spring-snapshots,spring-milestones
+      scs-bom:
+        groupId: io.pivotal.spring.cloud
+        artifactId: spring-cloud-services-starter-parent
+        mappings:
+          - versionRange: "[1.2.6.RELEASE,1.2.8.RELEASE]"
```

Comment:
Like this, with the paren? `[1.2.6.RELEASE,1.2.8.RELEASE)` Does that still restrict to Spring Boot 1.2?